geoapps/utils/plotting.py

```diff
@@ -541,38 +542,73 @@ def check_data_type(data):
     return data
 
 
-def plot_convergence_curve(h5file):
-    """"""
-    workspace = Workspace(h5file)
-    names = [group.name for group in workspace.groups if isinstance(group, SimPEGGroup)]
-    objects = widgets.Dropdown(
+def plot_convergence_curve(outfile: str) -> widgets.interactive | None:
+    """
+    Plot the convergence curve from a SimPEG *.out file.
+
+    :param outfile: Full path to the SimPEG *.out file
+
+    :return fig: widgets.interactive figure.
+    """
+
+    if not path.exists(outfile):
+        warnings.warn(f"File {outfile} does not exist.")
+        return
+
+    with open(outfile, encoding="utf-8") as f:
+        lines = f.readlines()
+
+    names = lines[0].split(" ")[:-1]
```

Comment:
why not just use np.genfromtxt here.  Seems like it should be flexible enough..

Reply:
Good call. I had tried `numpy.loadtxt` and that failed because of the time-date column. Will switch over.

Reply:
genfromtxt should have a usecols to skip the dates

Reply:
Done